crowdsourcing/viewsets/task.py

```diff
@@ -59,9 +61,15 @@ def retrieve_with_data(self, request, *args, **kwargs):
         requester_alias = task.project.owner.alias
         project = task.project.id
         target = task.project.owner.profile.id
+        timeout = task.project.timeout
+        worker_timestamp = task_worker.created_timestamp
+        now = datetime.datetime.utcnow().replace(tzinfo=utc)
+        timeout = 4 * 3600  # Default value in Seconds
```

Comment:
assume this is in minutes and task.project.timeout will return you a value 
i.e `timeout = models.Integerfield(default=240)` and then you can update line 67 here